### PR TITLE
SSKW 101 - Patient Home Page 

### LINKED ISSUE #101: SSKW - Patient Home Page
## Checklist ✅

- [x] The issue is present in the latest release.
- [x] I have searched the [issues](https://github.com/Sphinx-Society/kaizen-web-client/issues) of this repository and believe that this is not a duplicate.

## Summary 💡

The patient home page is the page the patients see on the home path.

## Requirements 🌈

- It must follow the design system
- It must have documentation
- It must have tests

## Motivation 🔦

To give the patient a page to watch the exams sent by the doctor.

### PR DESCRIPTION
**Pull request by omarefg**
## Checklist ✅

- [x] I have followed the [contributor guideline](https://github.com/Sphinx-Society/kaizen-web-client/blob/master/CONTRIBUTING.md)
- [ ] Tests are included
- [x] Documentation is changed or added
- [x] Related issue has been created
- [x] Commits messages follows [commit guideline](https://github.com/Sphinx-Society/kaizen-web-client/blob/master/CONTRIBUTING.md/#Commits)

resolves #101

### CODE REVIEW

File: src/client/components/pages/TestsHistory/TestsHistory.jsx [hunk 1]
```diff
@@ -6,41 +6,54 @@ import {
...
+    } else {
+      setFilteredTests(user.tests);
     }
   };
 
+  const editTe = () => {};
+
   useEffect(() => {
-    dispatch(listExams());
+    dispatch(getUser('5ec5f50ea76d8d1b91de94e5'));
```
Comment (jhnemogap) [inline src/client/components/pages/TestsHistory/TestsHistory.jsx:49]: This is the burnt text?

File: src/client/components/pages/TestsHistory/TestsHistory.jsx [hunk 2]
```diff
@@ -78,20 +93,14 @@ const ExamsHistory = (props) => {
                 id: 1,
               },
               {
-                header: 'Categoría',
-                accessor: 'type',
+                header: 'Doctor que receta',
```
Comment (jhnemogap) [inline src/client/components/pages/TestsHistory/TestsHistory.jsx:96]: **Assigned by doctor**
in spanish:
**Asignado por el doctor:**
dado que es un exámen no un jarabe

File: src/client/components/pages/TestsHistory/TestsHistory.jsx (hunk 1 above)
Comment (omarefg) [reply src/client/components/pages/TestsHistory/TestsHistory.jsx:49]: Hahahaah sorry, it  must be the hour. I'll fix that.

File: src/client/components/pages/TestsHistory/TestsHistory.jsx (hunk 2 above)
Comment (omarefg) [reply src/client/components/pages/TestsHistory/TestsHistory.jsx:96]: Hahahaa ok. I'll change that too.

**Comment by omarefg** [comment]
> Your PR #109
> and this PR #110
> They conflict

Well, it's ok. The second one that pass will have to resolve the conflicts

File: src/client/components/pages/TestsHistory/TestsHistory.jsx (hunk 1 above)
Comment (omarefg) [reply src/client/components/pages/TestsHistory/TestsHistory.jsx]: Done :grin:

File: src/client/components/pages/TestsHistory/TestsHistory.jsx (hunk 2 above)
Comment (omarefg) [reply src/client/components/pages/TestsHistory/TestsHistory.jsx]: Done :rocket:

**Pull request merged**